rkt/image/fetchers/docker/main.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	docker2aci "github.com/appc/docker2aci/lib"
+	d2acommon "github.com/appc/docker2aci/lib/common"
+	"github.com/coreos/rkt/rkt/image/fetchers"
+)
+
+func errAndExit(format string, a ...interface{}) {
+	out := fmt.Sprintf(format, a...)
+	fmt.Fprintln(os.Stderr, strings.TrimSuffix(out, "\n"))
+	os.Exit(1)
+}
+
+func main() {
+	config, err := fetchers.GetConfigFromStdin()
+	if err != nil {
+		errAndExit("error reading config: %v", err)
+	}
+
+	if !config.InsecureOpts.SkipImageCheck {
+		errAndExit("signature verification for docker images is not supported (try --insecure-options=image)")
```

Comment:
A fetcher plugin should not assume it is being used in rkt, I'd remove any references to rkt subcommand/option syntax like `(try --insecure-options=image)`.